index.html

```diff
@@ -97,7 +98,19 @@ <h2>
 
 <div class="mp-section">
   <div>
-    <h2><a href="team/">Who we are: our team</a></h2>
+    <h2>
+      <a href="/codebases/">
+        Where we work: our codebases
+      </a>
+    </h2>
+    <p>These public codebases improve citizens' lives and experience of government every day.</p>
+    <p><a href="/codebases/">We're proud to work with these codebase communities.</a></p>
+  </div>
+
```

Comment:
Is there a `</div>` missing here?

(Also, empty white lines)

Reply:
There's also an svg missing.

Reply:
I think I've fixed this now.